client/state/plans/actions.js

```diff
@@ -54,28 +44,10 @@ export const plansRequestFailureAction = error => {
 };
 
 /**
- * Fetch WordPress.com plans
+ * Action creator to request WordPress.com plans: REQUEST
  *
- * @return {Promise} Promise
+ * @return {Object} action object
  */
-export const requestPlans = () => {
-	return dispatch => {
-		dispatch( plansRequestAction() );
-
-		return wpcom
-			.withLocale()
-			.plans()
-			.list( { apiVersion: '1.4' } )
-			.then( data => {
-				dispatch( plansRequestSuccessAction() );
-				dispatch( plansReceiveAction( data ) );
-			} )
-			.catch( ( error ) => {
-				const message = error instanceof Error
-					? error.message
-					: error;
-
-				dispatch( plansRequestFailureAction( message ) );
-			} );
-	};
-};
+export const requestPlans = () => ( {
+	type: PLANS_REQUEST
```

Comment:
Do we want this to be a convention for naming of action types which we expect to trigger network requests, i.e. the `_REQUEST` suffix?